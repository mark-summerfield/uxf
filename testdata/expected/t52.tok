TClassBegin ttype=<Pair>
  Field Str("first") vtype=int
  Field Str("second") vtype=int
TClassEnd
TableBegin ttype=<Pair>
TableEnd
Eof
----------------------------------------
