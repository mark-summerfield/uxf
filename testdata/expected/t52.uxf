uxf 1
=Pair first:int second:int
(Pair)
