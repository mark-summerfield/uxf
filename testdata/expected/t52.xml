<?xml version="1.0" ?>
<uxf version="1.0">
  <ttypes>
    <ttype name="Pair">
      <field name="first" vtype="int"/>
      <field name="second" vtype="int"/>
    </ttype>
  </ttypes>
  <table name="Pair"/>
</uxf>
